packages/slate-editor/src/extensions/button-block/lib/createButtonBlock.ts

```diff
@@ -0,0 +1,27 @@
+import { v4 as generateUuid } from 'uuid';
+
+import { ButtonBlockNode } from '../ButtonBlockNode';
+
+export function createButtonBlock(
+    props: Partial<Omit<ButtonBlockNode, 'type' | 'children'>>,
+): ButtonBlockNode {
+    const {
+        href = '',
+        new_tab = false,
```

Comment:
I've checked other similar places (web bookmark, story bookmark) and we have `new_tab` set to `true` by default.

https://github.com/prezly/slate/blob/d83859080b95f1da7158cf775544d50cb3483239/packages/slate-editor/src/extensions/web-bookmark/lib/createWebBookmark.ts#L16-L26

Let's change it to `true` here as well.

Reply:
Updated